src/transactions.ts

```diff
@@ -6,7 +6,12 @@ const DEFAULT_GOVERNOR_PROXY = "0x2d56dC077072B53571b8252008C60e945108c75a";
 
 export const loadProposal = async (path: string) => {
   const proposalModule = await import(`../vips/${path}`);
-  return await proposalModule.default();
+  const proposal = await proposalModule.default();
+
+  delete proposal.gasFeeMultiplicationFactor;
+  delete proposal.gasLimitMultiplicationFactor;
```

Comment:
Instead of deleting specific fields, would it make sense to collect the fields we expect? That way, if we add new fields to the `proposal` object in the future, they won't be added by default. wdyt?